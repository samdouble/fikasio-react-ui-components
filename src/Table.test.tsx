import React from 'react';
import renderer from 'react-test-renderer';
import { Table } from '.';

describe('Table', () => {
  it('Renders correctly', () => {
    const componentJSON = renderer.create(<Table />).toJSON();
    expect(componentJSON).toMatchSnapshot();
  });
});
