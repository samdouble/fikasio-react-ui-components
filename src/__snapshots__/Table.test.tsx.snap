// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Renders correctly 1`] = `
<div
  className="table-responsive"
>
  <table
    className="fikasio-table fikasio-theme-dark table table-bordered table-hover"
    style={Object {}}
  >
    <thead>
      <tr />
    </thead>
    <tbody />
  </table>
</div>
`;
